README.md

```diff
@@ -237,14 +238,24 @@ Represents an absolute point in time, with an associated time zone.
 
 ### Constructor
 ```js
-new ZonedInstant(instant, timeZone)
+new ZonedDateTime(instant, timeZone)
 ```
 
 ### Properties
 ```js
-let milliseconds = zonedInstant.milliseconds;
-let nanoseconds = zonedInstant.nanoseconds;
-let timeZone = zonedInstant.timeZone;
+let year = zonedInstant.year;
+let month = zonedInstant.month;
+let day = zonedInstant.day;
+let hour = zonedInstant.hour;
+let minute = zonedInstant.minute;
+let second = zonedInstant.second;
+let millisecond = zonedInstant.millisecond; // 0-999
+let microsecond = zonedInstant.microsecond; // 0-999
+let nanosecond = zonedInstant.nanosecond; // 0-999
```

Comment:
I think it would be very confusing for civil values and instant values to have such similarly-named properties with such wildly different semantics.

Unit | CivilTime and CivilDateTime | ZonedDateTime | Instant
---- | ---- | ---- | ----
millisecond | whole milliseconds since last second transition | whole milliseconds since last second transition | whole POSIX milliseconds since epoch
microsecond | _absent_ | whole microseconds since last millisecond transition | whole POSIX microseconds since epoch
nanosecond | whole nanoseconds since last millisecond transition | whole nanoseconds since last _microsecond_ transition | whole POSIX nanoseconds since epoch